doctors_surgery.html

```diff
@@ -84,51 +84,49 @@ <h2>Need advice or have some feedback for us?</h2>
 
         <p>Send us a message by filling out the form below and one of our Receptionists will be in touch:</p>
 
-        <form action="" method="get">
+        <form>
```

Comment:
As you have an input field with a type of submit, you don't need to add an action here as it will be overridden by the input... I think?!